index.html

```diff
@@ -0,0 +1,73 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>MrBlue</title>
+	<link rel="stylesheet" href="static/style.css">
+	<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+</head>
+<body>
+	<header class="header">
+		<img src="static/logo.png" alt="" srcset="">
+     
+      <nav class="menu" id="menu">    
+        <div class="menu-close">
+            <a href="#" class="menu-close-icon"><img src="static/ic_menu.svg" alt="" srcset=""></a>
+        </div>
+        </nav>
+
+    </header>
+
+	<main>
+        <section class="bio">
+            <p class="bio-header">
+                Bleu Yves <br> Sopoude<br>
+                
+	
+            </p>
+            <p class="bio-info">
+                
+                Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br>  <br>
+
+                If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            
+            </p>
+
+			<p class="bio-info">
+                
+            </p>
+			<div class="link">
+                <p><button class="link-text" >Start collaboration</button></p>
```

Comment:
It's not necessary to enclose the `button` tag with a `p` tag. I think that you can remove the enclosing p tag.